lib/features/support/docker.rb

```diff
@@ -92,6 +92,32 @@ def run_docker_compose_command(file, command, must_pass=true)
   run_command(@script_env || {}, command, must_pass: must_pass)
 end
 
+
+def port_open?(ip, port, seconds=1)
+  Timeout::timeout(seconds) do
+    begin
+      TCPSocket.new(ip, port).close
+      true
+    rescue Errno::ECONNREFUSED, Errno::EHOSTUNREACH
+      false
+    end
+  end
+rescue Timeout::Error
+  false
+end
+
+def wait_for_port(port)
```

Comment:
Where is this used?

Reply:
But why doesnt it have an accompanying step?

Reply:
I think it just got missed, made a PR with the corresponding step https://github.com/bugsnag/maze-runner/pull/49